projects/extensions/datetimepicker/month-view.html

```diff
@@ -1,17 +1,19 @@
 <table class="mtx-calendar-table" role="grid">
   <thead class="mtx-calendar-table-header">
     <tr>
-      @for (day of _weekdays; track day) {
-        <th [attr.aria-label]="day.long">{{day.narrow}}</th>
+      @for (day of _weekdays; track day.long) {
+      <th [attr.aria-label]="day.long">{{day.narrow}}</th>
       }
     </tr>
   </thead>
-  <tbody mtx-calendar-body
+  <tbody
+    mtx-calendar-body
     (@slideCalendar.done)="_calendarStateDone()"
     [@slideCalendar]="_calendarState"
     [rows]="_weeks"
     [todayValue]="_todayDate!"
-    [activeCell]="_adapter.getDate(activeDate) - 1"
+    [activeCell]="_getActiveCell()"
```

Comment:
The active cell logic has no issue, we just should improve the a11y.
https://github.com/angular/components/blob/main/src/material/datepicker/month-view.html#L25

Reply:
Yes, I figured it out! Material has implemented a11y very effectively.